applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -803,13 +829,7 @@ play_messages([H|T]=Messages, PrevMessages, Count, #mailbox{timezone=Timezone
     AccountId = kapps_call:account_id(Call),
     Message = kvm_message:media_url(AccountId, H),
     lager:info("playing mailbox message ~p (~s)", [Count, Message]),
-    Prompt = [{'prompt', <<"vm-message_number">>}
-             ,{'say', kz_term:to_binary(Count - length(Messages) + 1), <<"number">>}
-             ,{'play', Message}
-             ,{'prompt', <<"vm-received">>}
-             ,{'say',  get_unix_epoch(kz_json:get_integer_value(<<"timestamp">>, H), Timezone), <<"current_date_time">>}
-             ,{'prompt', <<"vm-message_menu">>}
-             ],
+    Prompt = message_prompt(Messages, Message, Count, Box),
```

Comment:
you need to remove `timezone=Timezone` since `Timezone` is not used within the function
```
 play_messages([H|T]=Messages, PrevMessages, Count, #mailbox{timezone=Timezone
                                                            }=Box, Call) ->
```

Reply:
My mistake i missed that. I have removed it now. Thanks